Form/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { PropTypes } from 'react'
+import Fieldset from '../Fieldset'
+import Field from '../Field'
+import Input from '../Input'
+import compose from '../lib/compose'
+import uncontrolled from '../decorators/uncontrolled'
+
+function Form ({
+  fields,
+  fieldType,
+  focus,
+  onBlur,
+  onChange,
+  onFocus,
+  values,
+  ...props
+}) {
+  const FieldType = fieldType === 'field'
+    ? Field
+    : Input
+
+  return <Fieldset>
```

Comment:
1) I find it weird that `Form` doesn't create a `form`. This is important because of the of the "principle of least surprise". For instance, I may wanna write a test that asserts that I can see a form, and my intuition would be that I can check for a `form` tag;

1.1) I find it odd that there's no onSubmit that gives me a serialized object containing all values. It's very likely one would like to use this component uncontrolled and only get the values on a submit. Actually, the default behaviour should be just like an html form IMO.